app/poros/obituary.rb

```diff
@@ -3,12 +3,27 @@ class Obituary
               :abstract,
               :web_url,
               :pub_date,
-              :word_count
+              :word_count,
+              :image
+
   def initialize(obituary_data)
+
     @headline = obituary_data[:headline]
     @abstract = obituary_data[:abstract]
     @web_url = obituary_data[:web_url]
     @pub_date = obituary_data[:pub_date]
     @word_count = obituary_data[:word_count]
+    @image = sort_images(obituary_data[:image])
+  end
+
+  def sort_images(data)
+
+    if data.empty?
+      nil
+    else
+      data.select do |entry|
+        entry[:legacy][:thumbnail] != nil
+      end.first[:legacy][:thumbnail]
+    end
```

Comment:
This is so brilliant and I am so grateful that you took the time to write this simple, elegant code which accomplishes something that would have taken me twice as many lines and been really janky!

Reply:
Yes, this looks great Eric!